### PR TITLE
Added platform specific slash

### PR DESCRIPTION
**Pull request by cemremengu**
Testing on windows is hard.

This looks ugly so feel free to close if you think it's unnecessary

### CODE REVIEW

**Comment by cemremengu** [comment]
image

File: test/route.js [hunk 1]
```diff
@@ -183,7 +184,7 @@ test('/documentation should redirect to /documentation/index.html', t => {
   }, (err, res) => {
     t.error(err)
     t.strictEqual(res.statusCode, 302)
-    t.strictEqual(res.headers['location'], '/documentation/index.html')
+    t.strictEqual(res.headers['location'], `${path('/', 'documentation', 'index.html')}`)
```
Comment (delvedor) [inline test/route.js]: Did you test this on Windows?
I'm not sure it will work.

Ps: usually we wait for at least one *LGTM* before merging a pr :)

**Comment by mcollina** [comment]
I think this has to be reverted. the snapshot file is automatically generated by TAP, so it should not updated manually.

Also, headers always use `/`.

File: test/route.js (hunk 1 above)
Comment (cemremengu) [reply test/route.js]: Right I rushed a bit, apologies for that and yes I am on windows :)

Why do you think that?

Comment (delvedor) [reply test/route.js]: Nevermind, read @mcollina's comment :D

**Comment by cemremengu** [comment]
I am sorry for this. Any other ideas how we can run tests on windows?

**Comment by mcollina** [comment]
what error are you getting?

**Comment by mcollina** [comment]
https://github.com/fastify/fastify-swagger/blob/master/routes.js#L10 is a bug and it should not use `path.join`. The library is in fact broken on windows.

**Comment by cemremengu** [comment]
Changing to `handler: (request, reply) => reply.redirect(`${fastify.basePath}/index.html`)` doubles the `/` in one test, rest is fine

```
  with routePrefix: '/' should redirect to /index.html
  not ok should be equal
    +++ found
    --- wanted
    -/index.html
    +//index.html
```

**Comment by mcollina** [comment]
that should be fixed

Il giorno ven 28 set 2018 alle 09:59 Cemre Mengu <notifications@github.com>
ha scritto:

**Comment by cemremengu** [comment]
#110

**Pull request merged**